packages/server/src/portal-sync/portal-sync.service.ts

```diff
@@ -246,6 +246,7 @@ export class PortalSyncService {
     dataset.portal = Promise.resolve(portal);
     dataset.categories = allCategories;
     dataset.downloads = resource.download_count;
+    dataset.isTest = resource.name.toLowerCase().includes("test") && resource.name.split(" ").length <= 3
```

Comment:
Please add a comment here explaining the reasoning behind this logic. That way in the future when it's time to expand it we can remember why it was written the way it was